packages/stripe-integration/src/stripe-upe/stripe-ocs-payment-strategy.ts

```diff
@@ -308,7 +311,7 @@ export default class StripeOCSPaymentStrategy implements PaymentStrategy {
     private async _processAdditionalAction(
         error: unknown,
         methodId: string,
-    ): Promise<PaymentIntegrationSelectors | never> {
+    ): Promise<PaymentIntegrationSelectors | undefined | never> {
```

Comment:
why do we need "undefined" here?

Reply:
Changed. `never` removed as TSLint rules required `undefined` here